### PR TITLE
signond: update to 8.60.

### PR DESCRIPTION
**Pull request by Chocimier**

### CODE REVIEW

File: srcpkgs/signond/template [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
+makedepends="qt5-devel"
+short_desc="Daemon providing SSO over D-Bus for the gSSO framework"
+maintainer="John Rowley <enterthevoid@codesector.co>"
+license="LGPL-2.1-only"
 homepage="https://gitlab.com/accounts-sso/signond"
-distfiles="${homepage}/repository/archive.tar.gz?ref=VERSION_${version}>${pkgname}-${version}.tar.gz"
-checksum=23f6658bf358cc09b858402ee0857db2a2809276625eb01673a0a77969740762
+distfiles="${homepage}/repository/archive.tar.gz?ref=VERSION_${version}>signond-${version}.tar.gz"
+checksum=a9cd90cf3e8b1a8bdbe5acf46c797a817092e77946c69fc3ed3a76930f3a76aa
+nocross=qmake
```
Comment (jnbr) [inline srcpkgs/signond/template]: Can you provide more details?

Comment (jnbr) [reply srcpkgs/signond/template]: Yes, I care.
Most templates with `build_style=qmake` add everything to `hostmakedepends` as well to fix this.
I think adding the headers to the include path is the better solution.

Comment (Chocimier) [reply srcpkgs/signond/template]: unbroken

Comment (jnbr) [reply srcpkgs/signond/template]: Thank you. Any chance to fix armv7?

Comment (Chocimier) [reply srcpkgs/signond/template]: It was a random failure. Now passed with no changes.

**Pull request merged**